lib/bugsnag/payload.ex

```diff
@@ -116,13 +119,32 @@ defmodule Bugsnag.Payload do
         %{
           file: file,
           lineNumber: line_number,
-          inProject: Regex.match?(~r/^(lib|web)/, file),
+          inProject: in_project_fn.({module, function, args, file}),
           method: Exception.format_mfa(module, function, args),
           code: get_file_contents(file, line_number)
         }
     end)
   end
 
+  defp get_in_project_fn(options) do
+    case fetch_option(options, :in_project, nil) do
+      func when is_function(func) ->
```

Comment:
there are three listed in documentation. I think this fourth version is not in README, did you mean to add it there?